src/App.js

```diff
@@ -1,16 +1,95 @@
 import React from 'react';
+// import Todo from './components/TodoComponents/Todo';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
+
+const todoArr = [
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
+
+
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super();
+    this.state = {
+      todoArrObject: todoArr,
+      task: "",
+      id: "",
+      completed: ""
+    };
+  }
+
+  addTodo = event => {
+    event.preventDefault();
+    // console.log(event.target);
+    // setStudent with new student at the end of the array
+    // .push() makes us happy, because it works, but react doesn't like it. .concat is an option that works
+    // React disallows push because it will mutate the array and could end with side effects.
+    const newTodo = {
```

Comment:
```suggestion
    task: this.state.inputText
      id: Date.now()
      completed: false
```
So this is the initialization of the object that you're going to push into your todoArr on state.
So you want these to have these specific properties when you push them in.